hapi-proto/src/main/proto/ScheduleCreate.proto

```diff
@@ -28,6 +28,27 @@ option java_multiple_files = true;
 import "BasicTypes.proto";
 import "Timestamp.proto";
 
+/*
+Create a new Scheduled Transaction. After the Scheduled Transaction is created, the Schedule ID for it is set in the receipt.
+
+Users are able to create the Scheduled Transactions without the need for them to sign the scheduled underlying transaction (f.e Account A creates Scheduled Transaction for Account B, C and D to sign)
+
+Creating immutable scheduled transaction: Schedule Transaction can be created as immutable if the adminKey is omitted. In this case no one is able to execute Schedule Delete operation and the Scheduled Transaction will either execute or expire
+Defining the Payer of the Transactions: By default the payer of the Scheduled Transaction is the Account that creates it in the first place. Users however have the option to create Scheduled Transactions with Payer set to any account. In that case, the specified Payer Account ID must provide his signature for the scheduled transaction as-well.
```

Comment:
`Defining the Payer of the Transactions: By default, the payer of the Scheduled Transaction is the Account that creates it in the first place. However, users have the option to create Scheduled Transactions with Payer set to any account. If a Payer is provided, its signature will be required in order the scheduled transaction to reach execution.
`